app/views/zizia/csv_imports/_form.html.erb

```diff
@@ -49,6 +49,14 @@
               </div>
             </div>
           </div>
+          <div class="col-md-4">
+            <div class="well well-lg">
+              <p> <b>Import Behavior</b>. Choose how you would like exisiting IDs to handleded. </p>
```

Comment:
I believe you mean to say "Choose how you would like existing IDs to be handled."